wpa_supplicant/config_ssid.h

```diff
@@ -860,6 +860,38 @@ struct wpa_ssid {
 	 */
 	int macsec_integ_only;
 
+	/**
+	 * macsec_ciphersuite - Determines the ciphersuite for MACsec secure session
+	 *
+	 * 0: GCM-AES-128 (default)
+	 * 1: GCM-AES-256
+	 * 2: GCM-AES-XPN-128
+	 * 3: GCM-AES-XPN-256
+	 */
+	int macsec_ciphersuite;
+
+	/**
+	 * macsec_conf_offset - Determines the confidentiality offset for MACsec secure session
+	 *
+	 * 0: CONFIDENTIALITY_NONE (default)
+	 * 1: CONFIDENTIALITY_OFFSET_0
+	 * 2: CONFIDENTIALITY_OFFSET_30
+	 * 3: CONFIDENTIALITY_OFFSET_50
```

Comment:
just curious, can we have arbitrary confidentiality offset?

Reply:
No, see 802.1X-2020.  Confidentiality offset is transmitted between peers via EAPOL MKA with the current fixed values.